frontend/src/lib/components/Map/Map.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useRef } from 'react'
+import { Map as RawMap, Marker } from 'maplibre-gl'
+import useResizeObserver from 'use-resize-observer'
+
+import 'maplibre-gl/dist/maplibre-gl.css'
+
+/** Latitude and longtitude in degrees (+lat is east, -lat is west, +lon is south, -lon is north). */
+export interface MapProps {
+    /** Coordinates to center the map on by default. */
+    center: [number, number]
+    /** Markers to show. */
+    markers?: Marker[]
+    /** Map container class names. */
+    className?: string
+    /** The map's MapLibre style. This must be a JSON object conforming to the schema described in the MapLibre Style Specification, or a URL to such JSON. */
+    mapLibreStyleUrl: string
```

Comment:
If the type is `string`, then I suppose this cannot be a JSON object as the docstring says, and only a URL?